test/Faker/Provider/en_SG/PersonTest.php

```diff
@@ -10,6 +10,96 @@
  */
 final class PersonTest extends TestCase
 {
+    public function testFirstNameMaleMalay()
+    {
+        self::assertNotEmpty(Person::firstNameMaleMalay());
```

Comment:
They will never be empty due to how interfacing works. Please assert actual values with seeding instead.